tempodb/backend/v1/v1.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package backend.v1;
+
+import "pkg/tempopb/tempo.proto";
+
+import "google/protobuf/timestamp.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+
+option go_package = "github.com/grafana/tempo/tempodb/backend/v1";
+
+message BlockMeta {
+    string version = 1;
+    string block_id = 2;
```

Comment:
I wonder if instead of string we could use "bytes"

A GUID is a 16 byte array, as a string we are using 36 characters, which is 36 bytes (36 * uint8)

Reply:
I went back and forth on this, but I removed the code that make me choose string here.  I'll test with bytes.

Reply:
I've pushed a commit for this.  Good call out.